src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -1972,8 +2211,8 @@ def lrms_config_hook(cls, name, cfg, lrms, logger):
                the DVM.
         """
 
-        dvm_command = cls._which('orte-dvm')
-        if not dvm_command:
+        dvm_cmd = cls._which('orte-dvm')
+        if not dvm_cmd:
```

Comment:
Uhm, why the rename?  Now `dvm_command` is not set further below...  I don't mind the shorter name at all, but then it needs to be changed everywhere...